pkg/controller/syncsetinstance/syncsetinstance_controller.go

```diff
@@ -269,13 +274,25 @@ func (r *ReconcileSyncSetInstance) Reconcile(request reconcile.Request) (reconci
 		return reconcile.Result{}, applyErr
 	}
 
-	reapplyDuration := ssiReApplyDuration(ssi)
+	reapplyDuration := ssiReApplyDuration(ssi, r.reapplyInterval)
 	return reconcile.Result{RequeueAfter: reapplyDuration}, nil
 }
 
+func getReApplyInterval(log log.FieldLogger) time.Duration {
+	if envReapplyInterval := os.Getenv(reapplyIntervalEnvKey); len(envReapplyInterval) > 0 {
```

Comment:
Let's take this opportunity to be consistent about whether we want to use "Reapply" or "ReApply". It looks like the function names use "ReApply" and the variable names use "Reapply". Let's pick one.